appendices/VK_EXT_external_memory_metal.adoc

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 The Khronos Group Inc.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+include::{generated}/meta/{refprefix}VK_EXT_external_memory_metal.adoc[]
+
+=== Other Extension Metadata
+
+*Last Modified Date*::
+    2024-07-18
+*IP Status*::
+    No known IP claims.
+*Contributors*::
+  - Aitor Camacho Larrondo, LunarG Inc.
+
+=== Description
+
+An application may wish to reference device memory in multiple Vulkan device instances, in multiple processes, and/or in Metal API.
+This extension enables an application to export non-Vulkan handles from Vulkan memory objects such that the underlying resources can
```

Comment:
s/non-Vulkan handles/Metal handles/

Reply:
The text says "export", but the extension supports import too.

Reply:
Let me know if the new wording works!

Reply:
So close! "Import" is misspelled as "impor". Other than the typo, the text looks good.